### ISSUE TITLE
Error when starting ALM with a tsconfig with only a files array (without an exclude array)

### ISSUE THREAD

**Issue by markvandenbrink** [alm-tools/alm#108]
Hello @basarat,

Just looked into your ALM project. Very interesting stuff :-)

I came accross a small issue. I have a tsconfig file which only has a "files" array (and some compiler options). That is a valid tsconfig, but ALM throws an error when starting it. If I add an empty exclude array, ALM starts with the warning `TSCONFIG: Error You cannot use both "files" and "exclude" in tsconfig.json` (which is correct).

See attached image for detailed error. `toExpand` is undefined.

Thanks for all your hard work!

![alm-tsconfig](https://cloud.githubusercontent.com/assets/7869636/15537034/87b0851e-2274-11e6-874d-16df4802e6bf.png)

Throws an error:
```json
{
    "compileOnSave": false,
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "newLine": "LF",
        "declaration": false,
        "noImplicitAny": true,
        "noEmitOnError": false,
        "removeComments": true,
        "sourceMap": false,
        "outDir": "./compiled/"
    },
    "files": [
        "./index.ts"
    ]
}
```

Doesn't throw an error, but is an invalid config:
```json
{
    "compileOnSave": false,
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "newLine": "LF",
        "declaration": false,
        "noImplicitAny": true,
        "noEmitOnError": false,
        "removeComments": true,
        "sourceMap": false,
        "outDir": "./compiled/"
    },
    "files": [
        "./index.ts"
    ],
    "exclude": []
}
```

**Comment by basarat** [comment]
Sorry about that. Released with `0.0.69`. Should be on NPM soon :rose:

**Comment by markvandenbrink** [comment]
Thanks for your quick reply and fix! Works like a charm!
